kafka/client/client.go

```diff
@@ -0,0 +1,85 @@
+package kafkasender
+
+import (
+	"context"
+	"log"
+
+	"github.com/Shopify/sarama"
+	"github.com/cloudevents/sdk-go/protocol/kafka_sarama/v2"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+)
+
+//CloudEventsClient is the method signature for Kafka message
+type CloudEventsClient interface {
+	KafkaMessage(event string, userID string, email string, role []string)
```

Comment:
The name and/or signature seem a little off to me - the function name is generic, but the signature is too specific (implies user events only).

Reply:
Yeah you are right. Thanks I need to work on naming. Would it make more sense to be more marketo specific for the function? Like MarketoKafkaMessage?

Reply:
Never mind. Addressed by making this function more general and extracted all specific user even details